vote.py

```diff
@@ -133,13 +133,14 @@ def askyesno(self, event):
 
     def question_result(self, event=None):
         """Return results of question"""
-        res = ''
-        for k, v in self.answers.iteritems():
-            res += '{0} -> {1}, '.format(v[0], v[1])
+        def f(item):
```

Comment:
@Adman really, a helper function for this one line? Could you please at least call it something different than `f` if your argument is that pretty much the whole python world is against `lambda`s?